lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Maximum likelihood\n",
+    "\n",
+    "Computing and combining the sufficient statistics, the maximum likelihood estmimators are as follows:\n",
+    "\n",
+    "$ \\hat{a} = -2.95 $\n",
+    "\n",
+    "$ \\hat{b} = 26.26 $\n",
+    "\n",
+    "> Why is the MLE $\\hat{b}$ not the same as the posterior median value for $b$?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Distributions of estimators\n",
+    "\n",
+    "* In frequentism, we think of the estimators having distributions, since each dataset that we imagine being drawn from the sampling distribution will produce one estimator. An ensemble of (hypothetical) datasets leads to a (hypothetical) distribution of estimators\n",
+    "\n",
+    "* One straightforward approximate way to estimate these distributions is to use the asymptotic normality property of MLEs, and associate a _Gaussian approximation to the likelihood_ with the Gaussian distribution for the MLEs we expect to see when averaging over datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## The distribution of the log likelihood\n",
+    "\n",
+    "* An estimator can be thought of as a _summary_ of the data, and so can the value of the value of the log likelihood evaluated at the estimators. \n",
+    "\n",
+    "* The distribution of the log likelihood itself over the hypothetical ensemble of datasets provides a route to a _confidence interval._\n",
+    "\n",
+    "* In our simple Gaussian likelihood example, and also in the large dataset limit, the (negative) log likelihood (our $\\chi^2$ statistic) follows a $\\chi^2$ distribution with the same number of degrees of freedom as the dimensionality of the parameter space. Integrating this distribution from 0 to some boundary $\\Delta \\chi^2_{D}$ defines a confidence region in parameter space."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Confidence regions from $\\chi^2$ distributions\n",
+    "\n",
+    "* For example: in 1D, the 68% confidence region is bounded by the contour at $\\chi^2_{\\rm min} + \\Delta \\chi^2_{D}$ where $\\Delta \\chi^2_{D} = 1$ in 1D, and $\\Delta \\chi^2_{D} = 2.30$ in 2D. \n",
+    "\n",
+    "* In the 1D case, the boundary of the 68% confidence interval lies 1 standard deviation (or \"1-sigma\") from the mean.\n",
+    "\n",
+    "> In general, $\\Delta \\chi^2_{D}$ can be computed from the $\\chi^2$ distribution \"inverse survival function\", e.g.  `scipy.stats.chi2.isf(1.0-0.683, D)`"
```

Comment:
Would be good to clarify the relationship between the ISF and the quantile function, which has been introduced before.

Also, use 68 or 68.3 consistently.

Reply:
Good - re-jigged to use the quantile function (which scipy calls the "percentage point function") instead.